README.md

```diff
@@ -700,6 +700,18 @@ raise_on_not_specified_shard_query true
 raise_on_not_specified_shard_update true
 ```
 
+## Run tests locally
+
+```sh
+docker compose up -d
+# rbenv shell 2.7.8
+bundle install
+bundle exec appraisal install
+export TEST_MYSQL_HOST=127.0.0.1
+export TEST_MYSQL_PORT=13306
+bundle exec appraisal rails6_0_6 rake spec
+```
+
```

Comment:
Please add bellow meens.

`Not use appraisal if your old version ruby.  ex) Under Ruby 2.2.x`

Reply:
Added annotation 🙏  5ea0751
![](https://github.com/mixigroup/activerecord-turntable/assets/117768/ee2ad7c0-80cc-4810-8672-19d3a8e067f8)

Reply:
As a Notice, it should suggest the opposite meaning.

Reply:
Corrected the wording of the annotation 🙏  7a5d3c52e08dd8de0e2c4dfabaac014920ebe1f9

Reply:
@hirocaster Could you please check it? 🙇

Reply:
Thank you for your great work!